### ISSUE TITLE
React component: Masthead - Search edit field- shift-tab key from search edit field focus is going to the navigation menu items.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#3898]
## Detailed description
On react canary 

Issue: 
1) When we do shift-tab key from search edit field focus is going to the navigation menu items- for example (Products & Solutions Services & Consulting, Learn & support, Explore more, Covid 19).
2) But these menu items are not visible when the focus reaches to it.
3) Menu items are operable thru keyboard.

Expected: 
When we do shift tab key from search edit field, because search edit field is open and we could not see the menu item focus should go to the IBM Logo

Note: Working fine on Web components.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/3fe0d802-a70f-40e2-8ce9-ff55692ba8d7)

**Comment by IgnacioBecerra** [comment]
Seems like this issue is solved.

**Comment by Praveen-sr** [comment]
@IgnacioBecerra Re tested this, working fine now.

CC @RobertaJHahn
